src/main/java/net/sf/jabref/gui/shared/OpenSharedDatabaseDialog.java

```diff
@@ -90,13 +90,16 @@ public OpenSharedDatabaseDialog(JabRefFrame frame) {
     }
 
     public void openSharedDatabase() {
+        setLoadingConnectButton(true);
+
         try {
             bibDatabaseContext.getDBSynchronizer().openSharedDatabase(connectionProperties);
             frame.addTab(bibDatabaseContext, true);
             setGlobalPrefs();
             bibDatabaseContext.getDBSynchronizer().registerListener(new SharedDatabaseUIManager(frame));
             frame.output(Localization.lang("Connection_to_%0_server_stablished.", connectionProperties.getType().toString()));
```

Comment:
Sorry, but I couldn't help to notice "stablished" which should be "established".